controllers/trips/tripController.js

```diff
@@ -36,6 +36,29 @@ const getTrips = async function (req, res) {
   }
 };
 
+const getJoinedTrips = async function (req, res) {
+  //using a table join, extract and send back username 
+  //and email of all users tied to the requested trip in users_trips table
+  try {
+   // extract userid from req.body
+    const { userid } = req.body;
+
+    // confirm userid is not undefined
+    if (!userid)
+      return res.status(500).json({ message: 'user id is not defined' });
+
+    // retrieve user email and name using users and users_trips join table
+    const data = await knex('users_trips')
+    .join('users', 'users.id', 'user_id' )
+    .select('users.email', 'users.username')
+    .where({user_id: userid});
```

Comment:
we'll want to select by trip_id rather than user_id since we're pulling all users tied to a specific trup